src/test/resources/expected.iml

```diff
@@ -52,7 +63,7 @@
     <orderEntry type="module-library" scope="RUNTIME">
       <library>
         <CLASSES>
-          <root url="jar://$USER_HOME$/dev/inversoft/savant/savant-dependency-management/test-deps/integration/org/savantbuild/test/integration-build/2.1.1-{integration}/integration-build-2.1.1-{integration}.jar!/" />
+          <root url="jar://$USER_HOME$/dev/savant_source/savant/savant-dependency-management/test-deps/integration/org/savantbuild/test/integration-build/2.1.1-{integration}/integration-build-2.1.1-{integration}.jar!/" />
```

Comment:
This is trading 1 home directory dependent location for another. I should go back and address this, even though it's just a test file.